developers/weaviate/modules/retriever-vectorizer-modules/text2vec-transformers.md

```diff
@@ -20,14 +20,13 @@ Key notes:
 - Pre-built `BERT`, `DilstBERT`, `RoBERTa`, `DilstilROBERTa`, etc. models available.
 - Can also use any publicly available model from the [Hugging Face model hub](https://huggingface.co/models).
 
-:::tip Significant GPU/CPU speed differences
-Transformer models run *much* faster with GPUs, even for inference (10x+ speeds typically).
+:::tip Do you have GPU acceleration?
 
-Without a GPU, import or `nearText` queries may become bottlenecks in production if using `text2vec-transformers`.
+Transformer model inference speeds are usually much faster with GPUs, typical results are 10x faster. If you have a GPU, use one of the GPU enabled modules.<br/><br/>If you use `text2vec-transformers` without GPU acceleration, imports or `nearText` queries may become bottlenecks. The ONNX-enabled images can use [ONNX Runtime](https://onnxruntime.ai/) for faster inference processing on standard CPUs. Look for the `-onnx` suffix in the image name.<br/><br/>Alternatively, consider one of the following options:
```

Comment:
Suggest:

```
Transformer model inference speeds are usually much (~10x) faster with GPUs. If you have a GPU, use one of the GPU enabled models.
<br/>

If you use `text2vec-transformers` without GPU acceleration, imports or `nearText` queries may become bottlenecks. The ONNX-enabled images can use [ONNX Runtime](https://onnxruntime.ai/) for faster inference processing on CPUs. Look for the `-onnx` suffix in the image name.
<br/>

Alternatively, consider one of the following options:
```

Other than the line breaks, this changes "modules" to "models". `text2vec-transformers` is the Weaviate module to help users make use of transformer-architecture deep learning models. So some of these models are CPU-optimised, whereas most will benefit significantly from GPUs.

Reply:
Updated